### ISSUE TITLE
Could you introduce more about model training?

### ISSUE THREAD

**Issue by LetsGoFir** [Luodian/Otter#142]
Thanks for your excellent work! Hope for more details on training schedule, like training time, gpu number, and so on.

**Comment by ZhangYuanhan-AI** [comment]
Hi,

We train our image version model on the 8 A100 GPU, with 3 epochs in one day.

We use AdamW optimizer, with 1% training steps as the warmup, and 1e-5 as the LR.

**Comment by LetsGoFir** [comment]
> Hi,
> 
> We train our image version model on the 8 A100 GPU, with 3 epochs in one day.
> 
> We use AdamW optimizer, with 1% training steps as the warmup, and 1e-5 as the LR.

Thanks for your prompt reply! Image version seems like for Table. 2, then how about the video version (for Fig. 6(c) ) on MIMIC-IT dataset?

**Comment by Luodian** [comment]
I can provide you a very detailed training configs inside [GIST](https://gist.github.com/Luodian/b7a7bd74b5dbc556de08d802a48b07b1).

I directly pasted it from wandb. Inside it you could see all params we have and the values we set (some of them may not actually used during training).

The training loss would be like the following.

![Uploading image.png…]()

**Comment by wjfwjfwjf** [comment]
> @LetsGoFir I can provide you a very detailed training configs inside [GIST](https://gist.github.com/Luodian/b7a7bd74b5dbc556de08d802a48b07b1).
> 
> I directly pasted it from wandb. Inside it you could see all params we have and the values we set (some of them may not actually used during training).
> 
> The training loss would be like the following.
> 
> ![image](https://user-images.githubusercontent.com/15847405/247927888-ec27c0e2-3f01-4f28-93be-48bae5778494.png)

hello,i want to know the detail configs, but I can not open the https://gist.github.com/Luodian/b7a7bd74b5dbc556de08d802a48b07b1,what should I do
